Makefile

```diff
@@ -454,7 +454,7 @@ setup-envtest: $(SETUP_ENVTEST) ## Set up envtest (download kubebuilder assets)
 
 .PHONY: test
 test: $(SETUP_ENVTEST) $(GOVC) ## Run unit tests
-	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" GOVC_BIN_PATH=$(GOVC) go test -v ./apis/... ./controllers/... ./pkg/... ./internal/... $(TEST_ARGS)
```

Comment:
Why is that needed?

Reply:
That is because `gomonkey` doesn't support inline functions, and during testing, we need to disable Go language's inline optimization by using the command-line argument `-gcflags=-l`.